lib/rules/sort-prop-types.js

```diff
@@ -215,6 +217,22 @@ module.exports = {
       }
     }
 
+    function handleFunctionComponent(node) {
+      const firstArg = node.params[0].typeAnnotation && node.params[0].typeAnnotation.typeAnnotation;
+      if (firstArg && firstArg.type === 'TSTypeReference') {
+        const propType = variableUtil.findVariableByName(context, firstArg.typeName.name)
+        || (typeAnnotations.get(firstArg.typeName.name)
+          && typeAnnotations.get(firstArg.typeName.name)[0]);
```

Comment:
```suggestion
          || (
            typeAnnotations.get(firstArg.typeName.name)
            && typeAnnotations.get(firstArg.typeName.name)[0]
          );
```
looks like || fallback still isn't covered by a test case?

Reply:
I am not sure why it comes as uncovered, this fallback is essentially the fix for previous test case failure which was happening for `node 8/9` and `eslint 6`

Reply:
ah, perhaps it's codecov maxing out on what it can merge 🤔

Reply:
Fixed it, all checks are passing now.